### ISSUE TITLE
Error when date comes from string "startDate and endDate must be JavaScript Date Objects"

### ISSUE THREAD

**Issue by juancpena** [EddyVerbruggen/Calendar-PhoneGap-Plugin#146]
Inputting date as string: 

`var startDate = new Date("January 25, 2015 09:00:00");`

But i get the error "Error: startDate and endDate must be JavaScript Date Objects"

Here is what the resulting code looks like:

`window.plugins.calendar.createEvent('ING Miami Marathon 2015',null,'Creado por App SDCorre','Sun Jan 25 2015 09:00:00 GMT-0500 (EST)','Mon Jan 26 2015 09:00:00 GMT-0500 (EST)',function (message) { alert('Success: ' + JSON.stringify(message)); },function (message) { alert('Error: ' + message); });`

Any idea what is wrong? 

Thanks in advance!

**Comment by EddyVerbruggen** [comment]
Your dates need to be Date objects, but you're invoking `createEvent` with a date String.

**Comment by juancpena** [comment]
Thanks, is there a way to convert a string to a usable object? Ive tried Date.parse() but get milliseconds. 

Thanks in advance

**Comment by EddyVerbruggen** [comment]
Sure, have a look: http://www.w3schools.com/jsref/jsref_obj_date.asp

**Comment by juancpena** [comment]
Hi Eddy,

I am creating a Date object, this is just how it outputs the date:

Mon Apr 13 2015 06:00:00 GMT-0400 (EDT)

What should the resulting o'clock code look like if not this:

window.plugins.calendar.createEvent('Punta Cana Marathon 2015',null,'Creado por App SDCorre','Sun Apr 12 2015 06:00:00 GMT-0400 (EDT)','Mon Apr 13 2015 06:00:00 GMT-0400 (EDT)',function (message) { alert('Success: ' + JSON.stringify(message)); },function (message) { alert('Error: ' + message); });

**Comment by taringbabi** [comment]
I am trying to develop a Calendar application for Android Platform phones,
using jQuery mobile with Phonegap build.. after the converting process end
, its didnt work .. "error loding page" showed..
I want to use jQuery full calendar in my application, I have searched in
many websites. But not able to find exact solution.
I know there must be some way around to use jQuery Full Calendar in jQuery
Mobile.
by seesing some examples I have tried to use the jQuery Full Calendar
Plugin but I was not able to call the fullCalendar() Method.
